bom/pom.xml

```diff
@@ -55,6 +55,11 @@
                 <artifactId>mutiny-test-utils</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.smallrye.reactive</groupId>
+                <artifactId>mutiny-maths</artifactId>
```

Comment:
I found "maths" sounds wrong to me, so I went looking, and turns out, it's yet another US/UK difference: https://www.dailywritingtips.com/math-or-maths/ Wow :-)

Reply:
You got it wrong, it's the French word :-D

Reply:
I can rename it to singular.

Reply:
`-math` :-)

Reply:
done.